### ISSUE TITLE
Issue with original Conbee since 0.7.8

### ISSUE THREAD

**Issue by aqtrans** [mozilla-iot/zigbee-adapter#126]
My 'original' Conbee I USB adapter has been non-functional since updating the Zigbee Adapter addon on my Things Gateway to 0.7.8 a few days ago. 

With some playing around, I have been able to downgrade to 0.7.6 and get my Conbee adapter working perfectly once again. 

On 0.7.8, all my devices show up as connected, but do not respond to any commands. 

I've been able to find the following in the logs when 0.7.8 initializes: 
```
Jun  2 19:03:10 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:10.908 INFO   : Loading add-on: zigbee-adapter#033[22m#033[39m
Jun  2 19:03:11 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:11.741 INFO   : zigbee: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3#033[22m#033[39m
Jun  2 19:03:11 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:11.795 INFO   : zigbee: Loading add-on for zigbee-adapter from /home/pi/.mozilla-iot/addons/zigbee-adapter#033[22m#033[39m
Jun  2 19:03:11 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:11.803 INFO   : zwave: node3 valueChanged: 3-37-1-0:Switch property: on = false#033[22m#033[39m
Jun  2 19:03:12 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:12.544 INFO   : zigbee: DEBUG config = ''#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.330 INFO   : zigbee: this.driver = DeconzDriver {}#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.333 INFO   : zigbee: DeconzDriver: Using serial port /dev/ttyUSB0#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.367 INFO   : zigbee: DeConz Firmware version: 260b0500#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.397 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.410 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.425 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.442 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.458 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.474 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.489 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.506 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.522 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.537 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.553 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.570 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.585 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.601 INFO   : zigbee: Frame Status: 0: SUCCESS#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[37m#033[2m2019-06-02 19:03:13.620 INFO   : zigbee: Unhandled Rejection#033[22m#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.623 ERROR  : zigbee: { AssertionError [ERR_ASSERTION]: Out of Original Buffer's Boundary#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.625 ERROR  : zigbee:     at BufferReader.MAKE_NEXT_READER.BufferReader.(anonymous function) [as nextUInt32LE] (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/buffer-reader/index.js:77:9)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.626 ERROR  : zigbee:     at Object.param_parser.(anonymous function) (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/deconz-api/lib/frame-parser.js:69:30)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.627 ERROR  : zigbee:     at Object.frame_parser.(anonymous function) (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/deconz-api/lib/frame-parser.js:198:32)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.628 ERROR  : zigbee:     at DeconzAPI.parseFrame (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/deconz-api/lib/deconz-api.js:145:31)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.630 ERROR  : zigbee:     at DeconzAPI.onPacket (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/deconz-api/lib/deconz-api.js:127:26)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.631 ERROR  : zigbee:     at emitTwo (events.js:126:13)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.632 ERROR  : zigbee:     at Parser.emit (events.js:214:7)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.634 ERROR  : zigbee:     at Parser.parseChunk (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/deconz-api/lib/slip.js:60:16)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.635 ERROR  : zigbee:     at DeconzAPI.parseRaw (/home/pi/.mozilla-iot/addons/zigbee-adapter/node_modules/deconz-api/lib/deconz-api.js:170:21)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.636 ERROR  : zigbee:     at SerialPort.DeconzDriver.serialPort.on (/home/pi/.mozilla-iot/addons/zigbee-adapter/deconz-driver.js:132:15)#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.637 ERROR  : zigbee:   generatedMessage: false,#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.638 ERROR  : zigbee:   name: 'AssertionError [ERR_ASSERTION]',#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.640 ERROR  : zigbee:   code: 'ERR_ASSERTION',#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.641 ERROR  : zigbee:   actual: false,#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.642 ERROR  : zigbee:   expected: true,#033[39m
Jun  2 19:03:13 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:13.643 ERROR  : zigbee:   operator: '==' }#033[39m
Jun  2 19:03:16 gateway run-app.sh[496]: #033[31m2019-06-02 19:03:16.610 ERROR  : zigbee: This version of ConBee doesn't support the watchdog#033[39m
```

Please let me know if I can help troubleshoot in any other way.

**Comment by dhylands** [comment]
I've fixed an issue related to that. I have the original ConBee as well, running the same firmware. For a quick fix, you can update the firmware on the ConBee dongle. See: https://github.com/dresden-elektronik/deconz-rest-plugin/releases and https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually

I've updated the zigbee adapter and should a release out later today which should work with the older firmware (it's working with my dongle).

**Comment by dhylands** [comment]
This should be fixed now in 0.7.9

**Comment by aqtrans** [comment]
Yep, 0.7.9 fixed it up for me, thank you!
And thanks for the heads up on the firmware, I'll definitely have to update it.
